README.md

```diff
@@ -144,45 +180,41 @@ _refinery_ consists of multiple services that need to be run together. To do so,
 If you run into any issues during installation, please don't hesitate to reach out to us (see community section below).
 
 ### Persisting data
+
 By default, we store the data to the directory `refinery/postgres-data`. If you want to change that path, you need to modify the variable `LOCAL_VOLUME` of the `start` script of your operating system. To remove data, simply delete the volume folder. **Make sure to delete only if you don't need the data any longer - this is irreversible!**
 
 ## 📘 Documentation and tutorials
-The best way to start with _refinery_ is our [**quick start**](https://docs.kern.ai/docs/quickstart).
 
-You can find extensive guides in our [README docs](https://docs.kern.ai/docs) and [tutorials](https://www.youtube.com/channel/UCru-6X24b76TRsL6KWMFEFg/videos) on our YouTube channel. We've also prepared a [repository with sample projects](https://github.com/code-kern-ai/sample-projects) which you can clone.
+The best way to start with _refinery_ is our [**quick start**](https://docs.kern.ai/refinery/quickstart).
+
+You can find extensive guides in our [docs](https://docs.kern.ai) and [tutorials](https://www.youtube.com/channel/UCru-6X24b76TRsL6KWMFEFg/videos) on our YouTube channel. We've also prepared a [repository with sample projects](https://github.com/code-kern-ai/sample-projects) which you can clone.
```

Comment:
i think i would have linked to https://www.youtube.com/@kern_ai/featured (and ensured that the home screen is a bit more optimized with playlists directly on the screen)

Reply:
will change this to https://www.youtube.com/@kern_ai/videos